src/components/input/input.scss

```diff
@@ -23,6 +23,10 @@ ion-textarea {
   width: 100%;
 }
 
+ion-input[disabled] .input-cover {
```

Comment:
Can you move the rule so it is with the other rule related to input cover: https://github.com/driftyco/ionic/blob/master/src/components/input/input.scss#L77-L94

you could add it after that rule and add a sentence in the description saying the input cover shouldn't be clickable when the input is disabled.

Otherwise it looks good, just make sure there is one line break between selectors and two before a header. :)